src/environment/SCRF.h

```diff
@@ -43,26 +54,29 @@ class SCRF final {
          int kain_hist,
          int max_iter,
          bool dyn_thrs,
-         const std::string &density_type);
+         SCRFDensityType density_type);
     ~SCRF();
 
     double setConvergenceThreshold(double prec);
 
-    mrcpp::ComplexFunction &getCurrentReactionPotential() { return this->Vr_n; }
-
     Permittivity &getPermittivity() { return this->epsilon; }
 
     void updateMOResidual(double const err_t) { this->mo_residual = err_t; }
 
-    friend class ReactionPotential;
     auto computeEnergies(const Density &rho_el) -> std::tuple<double, double>;
```

Comment:
Since you use `std::tie` as the return statement in `FockBuilder`, should we also just write this as 
```suggestion
    auto computeEnergies(const Density &rho_el) -> std::tie<double, double>;
```
to keep it clear what we are working with?

Reply:
No, `tie` is to bind elements of a tuple to existing variables: it's a function not a data container. https://en.cppreference.com/w/cpp/utility/tuple/tie

Reply:
alright